Package.swift

```diff
@@ -37,7 +37,9 @@ let package = Package(
     ),
     .target(
       name: "PovioKitUI",
-      dependencies: [],
+      dependencies: [
+        "PovioKit"
+      ],
```

Comment:
Why do we need this dependency?

Reply:
I'm using `UIColor` extension and `Builder` class of `PovioKit`. In fact I started using them before we split the UI component on its own. 

We can replace these auxiliary with "system" methods for both, to get rid of the dependency, what do you think?

Reply:
Ah ok. That's fine. It would be a strange case to only use `PovioKitUI` in the project.